### ISSUE TITLE
mAp stuck a low value for vgg16

### ISSUE THREAD

**Issue by datNurd** [fizyr/keras-retinanet#924]
I am trying to train a custom object detection model. I've train the data set on retinanet with Resnet50 as backbone and achieved a mAp of 60, but there was a problem of overfitting since my data set is small about 1500 image approx. So I wanted to train on less complex/shallow backed i.e vgg16 but as I see the loss is decreasing(not so drastically) and mAp is stuck at 0.0168. 
I am using the imagenet weights from keras.

**Comment by datNurd** [comment]
Sorry made a mistake

**Comment by dilpreetsingh** [comment]
Hey man, I'm getting similar behaviour, what ended up being your issue?
